_episodes/00-intro.md

```diff
@@ -1,5 +1,17 @@
 ---
-title: "Introducing high-performance computing"
+title: "Introducing High-Performance Computing"
 questions:
-- Why use a high-performance computing cluster?
+- Why use a large-scale computing system?
 ---
+
+Topics
+
+When it is appropriate to use a cluster
```

Comment:
I would also add:

How is using a large-scale computing system different from using your laptop? Talk to your neighbor about some differences you may already know about, and some differences/difficulties you imagine you may run into.